tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
+
+[ -z "${KUBECONFIG}" ] && echo "KUBECONFIG is not set" && exit 1
+
+oc create namespace kubevirt
+rm -rf _out
+hack/dockerized hack/build-func-tests.sh
+make manifests
+rm -f _out/manifests/testing/cdi-*
+rm -f _out/manifests/testing/kubevirt-config.yaml
+sed -i "s/namespace: kubevirt/namespace: ${KUBEVIRTNAMESPACE}/g" _out/manifests/testing/*.yaml
+_out/tests/tests.test --cdi-namespace=${KUBEVIRTNAMESPACE} \
+	--deploy-testing-infra \
+	--ginkgo.seed=0 \
```

Comment:
lets use 42 as prow use ?

Reply:
Why? This is in practice disables the permutation, why 42 is better than 0?

Reply:
just to run in the order that prow runs, so the flow will be as close as

Reply:
Why do we need the flow to be in sync with the CI? This is a manual run, on a different setup with probably different OS/s and deployments.
The only requirement is for these to be consistent between runs on the same setup, nothing more.

Reply:
42 has specific order on the CI, 
and if someone would use it locally and want to reproduce an error that happened on CI,
this way it would be as close to the CI ,and increase the changes that it will save him time, and that it might happen,

imo both should be the same, even if on CI we change to 0 , which is more native